test/configs/webpack.config.js

```diff
@@ -47,31 +47,32 @@ module.exports = async (env, options) => {
         },
         plugins: [
             new CleanWebpackPlugin({
-                cleanOnceBeforeBuildPatterns: dev ? [] : ["**/*"]
+                cleanOnceBeforeBuildPatterns: dev ? [] : ["**/*"],
+                cleanAfterEveryBuildPatterns: dev ? ["!**/*"] : []
```

Comment:
Could you explain the reason to add cleanAfterEveryBuildPatterns?

Reply:
We had added this for the normal execution of Excel CFs, because without it the project doesn't build properly (it deletes some HTML files).
I thought it would be nice to also add this to the test files

Reply:
OK